target/linux/mediatek/dts/mt7981b-xiaomi-mi-router-common.dtsi

```diff
@@ -83,6 +83,14 @@
 		interrupt-parent = <&pio>;
 		interrupts = <38 IRQ_TYPE_LEVEL_HIGH>;
 	};
+
+	switch2: switch@1 {
+		compatible = "airoha,an8855";
+		reg = <1>;
+		reset-gpios = <&pio 39 GPIO_ACTIVE_HIGH>;
+		airoha,ext_surge;
```

Comment:
this should be airoha,ext-surge. Probably the reason why the port didn't work for some? (aka they weren't calibrated?)

Reply:
Done